pkgpanda/__init__.py

```diff
@@ -261,6 +270,19 @@ def validate_compatible(packages, roles):
                         k, v, package))
             environment[k] = package
 
+        # No conflicting sysctl values.
+        for service_name, sysctl_settings in package.sysctl.items():
+            for sysctl_var, sysctl_value in sysctl_settings.items():
+                if sysctl_map.get(sysctl_var, None) == sysctl_value:
```

Comment:
Shouldn't this be `!=`? As written isn't it checking that everything in sysctl_settings matches what is in sysctl_map?